src/clr-angular/utils/focus/key-focus/key-focus.spec.ts

```diff
@@ -208,15 +212,15 @@ describe('KeyFocus directive', () => {
         openMenu();
         expect(document.activeElement.textContent).toBe('Button 1');
         // Arrow down
-        keyPress('ArrowDown');
+        keyPress(KeyCodes.ArrowDown);
         expect(document.activeElement.textContent).toBe('Button 2');
         // Make sure it's only ArrowDown/Up
-        keyPress('ArrowRight');
+        keyPress(KeyCodes.ArrowRight);
         expect(document.activeElement.textContent).toBe('Button 2');
-        keyPress('ArrowLeft');
+        keyPress(KeyCodes.ArrowLeft);
         expect(document.activeElement.textContent).toBe('Button 2');
         // Arrow up
-        keyPress('ArrowUp');
+        keyPress(KeyCodes.ArrowUp);
```

Comment:
Wanted to remove magic strings, I caught myself making a couple typos from the key codes.